docs/reference/api/README.md

```diff
@@ -330,6 +359,43 @@ you `DELETE /api/function_invocations`, the HTTP method in the event as received
 ...Function replica logs...
 ```
 
+### Proxy logs from a source
+
+#### Request
+
+* URL: `GET /api/functions/<function name>/logs/proxy-logs`
```

Comment:
in `function.go` the endpoint is defined as `/api/functions/<function name>/proxy-logs`